### PR TITLE
refactor: use framework.ExpectNoError instead

### PR DESCRIPTION
**Pull request by draveness**
**What type of PR is this?**
 /kind cleanup
/priority backlog
/sig testing

**What this PR does / why we need it**:

Ref: https://github.com/kubernetes/kubernetes/issues/77103

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: test/e2e/upgrades/apps/statefulset.go [hunk 1]
```diff
@@ -18,7 +18,6 @@ package upgrades
 
 import (
 	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
```
Comment (oomichi) [inline test/e2e/upgrades/apps/statefulset.go]: I love to see this removal :-)

File: test/e2e/upgrades/storage/persistent_volumes.go [hunk 2]
```diff
@@ -23,7 +23,6 @@ import (
 	"k8s.io/kubernetes/test/e2e/framework/volume"
 
 	"github.com/onsi/ginkgo"
-	"github.com/onsi/gomega"
```
Comment (oomichi) [inline test/e2e/upgrades/storage/persistent_volumes.go]: ++

**Review by oomichi — COMMENTED** [review]
@draveness Thank you so much for this work!

/lgtm

**Comment by draveness** [comment]
@oomichi Rebased, PTAL : D

**Review by oomichi — COMMENTED** [review]
/lgtm

**Review by timothysc — APPROVED** [review]
/approve 

thx @draveness !

**Comment by oomichi** [comment]
`The old job viewer, Gubernator, has been deprecated in favour of this page, Spyglass. For now, the old page is still available. Please send feedback to sig-testing. ` on the job dashboard. Let's retry.

/retest

**Pull request merged**